.gitignore

```diff
@@ -1,3 +1,5 @@
+node_modules/
+
 .DS_Store
 
-node_modules
+index.js
```

Comment:
We should probably try to find a way to handle complied JS files as we go as we will definitely have more than one in the future... Maybe we should have them output in the `dist` folder and then add the `dist` folder to the gitignore?